infra/core/signalr/signalr.bicep

```diff
@@ -0,0 +1,36 @@
+metadata description = 'Creates an Azure SignalR Services instance.'
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+@description('Controls whether local authentication is disabled')
+param disableLocalAuth bool = true
+
+@description('The SKU of the SignalR service')
+param sku object = {
+  name: 'Premium_P1'
+}
+
+resource signalR 'Microsoft.SignalRService/signalR@2023-08-01-preview' = {
+  name: name
+  location: location
+  tags: tags
+  sku: sku
+  identity: {
```

Comment:
This can also be user assigned identity. I saw web.bicep has such a identity. Should we use that?